lib/Core/Executor.cpp

```diff
@@ -436,16 +436,27 @@ const char *Executor::TerminateReasonNames[] = {
   [ Unhandled ] = "xxx",
 };
 
-
-Executor::Executor(LLVMContext &ctx, const InterpreterOptions &opts,
-                   InterpreterHandler *ih)
+static int tries = 0;
+bool enteredDevMain = false;
+Executor::Executor(LLVMContext &ctx,
+	           const InterpreterOptions &opts,
+                   InterpreterHandler *ih,
+		   						 TimingSolver* s,
```

Comment:
be consistent with naming. Some variables start with _, some don't.

Reply:
Makes sense. I've removed the '_' and initialized the variables using the initializer list.